conf/development.ini

```diff
@@ -29,10 +29,6 @@ h.feature.notification: False
 h.feature.queue: False
 h.feature.streamer: False
 
-horus.register_redirect: stream
-
-horus.require_activation: False
```

Comment:
Removing differences between dev and production, removing unnecessary configurations - I like this